addons/website_sale/static/src/js/website_sale.editor.js

```diff
@@ -376,7 +376,8 @@ options.registry.WebsiteSaleProductsItem = options.Class.extend({
         $ribbons.removeClass(htmlClasses);
 
         $ribbons.addClass(ribbon.html_class || '');
-        $ribbons.attr('style', `background-color: ${ribbon.bg_color || ''} !important`);
+        $ribbons.attr('style',
+            `background-color: ${ribbon.bg_color ? `${ribbon.bg_color} !important` : 'inherit'}`);
```

Comment:
why do you need `'inherit'`?

Reply:
If 'ribbon.bg_color' is not provided(false), the background color of the '$ribbon' element is set to **'inherit'** so it will take the background color of its parent element. This will maintain a consistent background color of the element.